rpadmin/api_broker.go

```diff
@@ -46,6 +47,9 @@ const (
 type Broker struct {
 	NodeID           int                `json:"node_id"`
 	NumCores         int                `json:"num_cores"`
+	Rack             string             `json:"rack"`
+	InternalAddress  string             `json:"internal_rpc_address"`
+	InternalPort     int                `json:"internal_rpc_port"`
```

Comment:
```suggestion
	InternalRPCAddress  string             `json:"internal_rpc_address"`
	InternalRPCPort     int                `json:"internal_rpc_port"`
```

Just to not confuse it with other addresses/ports